frontend/src/app/okrview/navigation-list-entry/navigation-list-entry.component.ts

```diff
@@ -1,53 +1,45 @@
-import { Component, Input, OnDestroy, OnInit } from '@angular/core';
-import { Subscription } from 'rxjs';
+import { Component, Input, OnInit } from '@angular/core';
 import { OkrUnitSchema, OkrUnitRole } from '../../shared/model/ui/okr-unit-schema';
 import { CurrentNavigationService } from '../current-navigation.service';
-import { DepartmentNavigationInformation } from '../../shared/model/ui/department-navigation-information';
 
 @Component({
   selector: 'app-navigation-list-entry',
   templateUrl: './navigation-list-entry.component.html',
   styleUrls: ['./navigation-list-entry.component.scss']
 })
-export class NavigationListEntryComponent implements OnInit, OnDestroy {
+export class NavigationListEntryComponent implements OnInit {
   @Input() schema: OkrUnitSchema;
   @Input() isSecondUnit: boolean = true;
-  @Input() startsOpen: boolean = false;
 
-  currentNavigationInformation = new DepartmentNavigationInformation(-1, []);
-  navigationInformationSubscription: Subscription;
-  isOpen = false;
+  isOpen = true;
 
   constructor(private currentNavigationService: CurrentNavigationService) {}
 
   ngOnInit(): void {
-    if (this.startsOpen) {
-      this.isOpen = true;
+    if(this.currentNavigationService.getClosedStructures().departmentsToClose.includes(this.schema.id)){
+      this.isOpen = false;
```

Comment:
die Logik aus der IF-Abfrage würde ich in eine Methode im currentNavigationService verlegen. Hier in der ngOnInit-Methode würde dann ein Aufruf genügen
```suggestion
    this.isOpen = this.currentNavigationService.isStructureMarkedAsClosed(this.schema.id);
```